application/src/main/java/org/opentripplanner/routing/algorithm/filterchain/ItineraryListFilterChain.java

```diff
@@ -6,21 +6,25 @@
 import org.opentripplanner.routing.algorithm.filterchain.framework.filterchain.DeleteResultHandler;
 import org.opentripplanner.routing.algorithm.filterchain.framework.filterchain.RoutingErrorsAttacher;
 import org.opentripplanner.routing.algorithm.filterchain.framework.spi.ItineraryListFilter;
+import org.opentripplanner.routing.algorithm.filterchain.paging.PageCursorInputAggregator;
```

Comment:
The PageCursorInputAggregator belong in the filterchain package.

Reply:
I moved it into the filterchain package